custom/enikshay/user_setup.py

```diff
@@ -168,6 +169,22 @@ def set_available_tests(location, location_form):
         location.metadata['tests_available'] = 'cbnaat'
 
 
+def save_user_callback(sender, couch_user, **kwargs):
+    if toggles.ENIKSHAY.enabled(couch_user.domain):
+        set_issuer_id(couch_user.domain, couch_user)
```

Comment:
I don't know, actually awaiting clarification on that.  One thing that's not obvious is that `couch_user` here will only ever be a `CommCareUser` (the signal is `commcare_user_post_save`).  I tried naming the parameter `commcare_user`, but Django requires that signal receivers have parameter names that match up with the `providing_args`.